src/features/journey/components/Results/RouteSegment.tsx

```
@@ -251,14 +352,77 @@ export function RouteSegment({
+                        <div key={idx} className="flex items-center py-1.5">
+                          <div
+                            className={`w-2 h-2 rounded-full ${
+                              isStartingPoint || isOrigin
+                                ? 'bg-emerald-500'
+                                : isEndPoint || isDestination
+                                  ? 'bg-red-500'
+                                  : 'bg-gray-300'
+                            } mr-2.5 flex-shrink-0`}
+                            aria-hidden="true"
+                          ></div>
+                          <div className="flex items-center flex-wrap gap-1.5">
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)